### DISCUSSION TITLE
Plotting stopped but no error code.

### DISCUSSION THREAD

**Discussion by justmemo** [Chia-Network/chia-blockchain#5474] [Support]
**Describe the bug**
I was trying plot process but i got the same situation every day. 
Plot stops in different percentages. Not the same percentages. No error code
What can i do for that's situation?   
99 Percentages waiting from 6 hours , WHY ? 

**Screenshots**

![WhatsApp Image 2021-05-18 at 8 29 14 AM](https://user-images.githubusercontent.com/84389361/118653311-636d4080-b7b5-11eb-8467-106dcf859e46.jpeg)

![WhatsApp Image 2021-05-18 at 8 29 14 AM (1)](https://user-images.githubusercontent.com/84389361/118653240-505a7080-b7b5-11eb-97cd-9c82448a37bc.jpeg)

**Comment by kushariforyou** [comment]
What settings are you using for the plots, and what drives are you writing to? Also what version of Chia are you running?

**Comment by justmemo** [comment]
**> What settings are you using for the plots, and what drives are you writing to? Also what version of Chia are you running?**
Usually, I'm using it like that. ( Screenshot)
Temporary Directory: Samsung 980 pro 1tb 
Equipment : 
Cpu: Intel i7 11700K
Ram: Corsair 16 gb 3200 Mhz
I have 1.1.15 version
 **Screenshot**
![WhatsApp Image 2021-05-18 at 10 02 55 AM](https://user-images.githubusercontent.com/84389361/118665863-ada7ef00-b7c0-11eb-8b5b-acedd1969d30.jpeg)

**Comment by kushariforyou** [comment]
Are you running low on space? I'd only do around 2 plots max for a 1TB drive at the same time.

**Comment by justmemo** [comment]
Maybe but I had more than 300 Gb of storage

**Comment by kushariforyou** [comment]
Try with 1 or 2 plots instead of 3 and see if that helps.

**Comment by justmemo** [comment]
im will try in a day and i will share update . Thanks

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
